docs/configuration/voice.md

```diff
@@ -1,29 +1,33 @@
-## Voice
+# Text to Speech
 
 Enter this command to use TTS _(Text-to-Speech)_ for Auto-GPT
 
 ``` shell
 python -m autogpt --speak
 ```
 
-Eleven Labs provides voice technologies such as voice design, speech synthesis, and premade voices that Auto-GPT can use for speech.
+Eleven Labs provides voice technologies such as voice design, speech synthesis, and
+premade voices that Auto-GPT can use for speech.
 
-1. Go to [Eleven Labs](https://beta.elevenlabs.io/) and make an account if you don't already have one.
+1. Go to [ElevenLabs](https://beta.elevenlabs.io/) and make an account if you don't
+    already have one.
 2. Choose and setup the `Starter` plan.
 3. Click the top right icon and find "Profile" to locate your API Key.
 
 In the `.env` file set:
+
 - `ELEVENLABS_API_KEY`
 - `ELEVENLABS_VOICE_1_ID` (example: _"premade/Adam"_)
 
-### List of IDs with names from eleven labs. You can use the name or ID:
```

Comment:
I think the note "You can use the name or ID" was pretty useful.
Also if people use the name, it's much more readable

Reply:
good one